charts/opentelemetry-operator/templates/admission-webhooks/mutatingwebhookconfiguration.yaml

```diff
@@ -0,0 +1,51 @@
+{{- if and .Values.admissionWebhooks.enabled }}
```

Comment:
```suggestion
{{- if .Values.admissionWebhooks.enabled }}
```

This seems out of place since there's only one operand.

Reply:
Yes, this is my mistake. Thank you for pointing out.